documentation/cobigen-intellij-installation.asciidoc

```diff
@@ -0,0 +1,67 @@
+:toc:
+toc::[]
+
+
+= Installation
+
+
+== IntelliJ setup
+
+If you want to use IntelliJ for CobiGen project, you can follow the steps below:
+
+. You can easily install IntelliJ, create a launch script for it etc. as described in
+
+`https://github.com/devonfw/ide/blob/master/documentation/intellij.asciidoc`
+
+. After starting IntelliJ, you **need** to import the cobigen folder as maven project
+
+. Next you **have** to configure the maven settings in IntelliJ as follows:
+
+.. Go to `File`-> `Settings` -> `Build, Execution, Deployment` -> `Build Tools` -> `Maven`
+
+... **Choose** `(Maven home Path)` from your devonFolder -> devonProjects/software/Maven
+
+... **Override** `(user settings file)` -> devonProjects/conf/.m2/settings.xml
+
+... **Override** `(Local repository)` -> devonProjects/conf/.m2/repository
+
+
+image::images/howtos/intellij/maven-settings.png[Maven Settings]
+
+.. Now go to `File` -> `Project structure` -> `Project Settings` -> `Project add java sdk version from` -> `devonProjects/software/java`
```

Comment:
We need to empathize, that only Java 11 will work here. The latest devonfw-ide uses Java 19 which will not work with this setup.
The user will have to download Java 11 manually and place it into devonProjects/software/java11.

Reply:
Can be ignored if the user uses the new settings URL from our devon-ide setup tutorial.
Devon will install Java17 then automatically.